server/src/main/java/org/opensearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -3050,19 +3117,22 @@ private void cacheRepositoryData(BytesReference updated, long generation) {
             try {
                 serialized = CompressorRegistry.defaultCompressor().compress(updated);
                 final int len = serialized.length();
-                if (len > ByteSizeUnit.KB.toBytes(500)) {
+                long cacheWarningThreshold = Math.min(repositoryDataCacheThreshold * 10, Integer.MAX_VALUE);
```

Comment:
less important for a warning but should we use the -8 here too?

Reply:
You’re right! I missed that part in my previous updates. Thank you for the careful feedback 👍
I’ve made the adjustments, pushed the changes, and the CI has passed 🎉😆

Reply:
You’re right! I missed that part in my previous updates. Thank you for the careful feedback 👍
I’ve made the adjustments, pushed the changes, and the gradle check has passed 🎉😆